pkgs/servers/roon-server/default.nix

```diff
@@ -9,16 +9,18 @@
 , makeWrapper
 , stdenv
 , zlib
-}: stdenv.mkDerivation rec {
+}:
+let
+  major = "1";
+  minor = "8";
+  build = "831";
+in
+stdenv.mkDerivation {
   pname = "roon-server";
-  version = "1.8-831";
+  version = "${major}.${minor}-${build}";
```

Comment:
This breaks @r-ryantm . The tarball name could be generated with `builtins.replaceStrings [ "." "-" ] [ "00" "00" ]`

Reply:
Nice catch, fixed :)